recipes/starfile/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "starfile" %}
+{% set version = "0.4.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/starfile-{{ version }}.tar.gz
+  sha256: 36f44a244fb891b604e44d31f20b68ffabaec3eb14c57f9eb66c2a5acf94427d
+
+build:
+  entry_points:
+    - starfile = starfile.__main__:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - pandas
+    - python >=3.8
+
+test:
+  imports:
+    - starfile
+  commands:
+    - pip list
+    - pip check
+    - starfile --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/alisterburt/starfile
+  summary: STAR file reading and writing in python.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - alisterburt
+    - conda-forge/napari
```

Comment:
This one does not depend on napari, but I assume is a dependency of a plugin. Do we still want the team to be maintaining this one too?

Reply:
Better not :)